cfpb/consumerfinance.gov
# Sticky hover state fix on mobile buttons

## Changes

-Adds a media query to fix "sticky state" on mobile search button and mega menu button


## How to test this PR

1. Click on search button, then click it again to close, notice no more hover state
2. Click on mega menu button, then click it again to close,  notice no more hover state


## Screenshots

Before:
![image](https://user-images.githubusercontent.com/85513449/149020176-f4d8f48f-e80f-4f82-a881-4ce450d39449.png)

![image](https://user-images.githubusercontent.com/85513449/149020331-5ea02304-ab7f-44c9-9757-d0b9c1e38650.png)


After:
![image](https://user-images.githubusercontent.com/85513449/149020254-ff653abd-6537-48a8-b60c-ecd86caf1267.png)

![image](https://user-images.githubusercontent.com/85513449/149020379-ef4973a6-e683-45f2-866c-c5ffed0af5a3.png)

Resolves 3930
